tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -169,9 +170,14 @@ public function addProductsToOrderWithNewInvoiceAndTheFollowingDetails(string $o
     {
         $orderId = SharedStorage::getStorage()->get($orderReference);
 
+        $taxCalculator = $this->getOrderTaxCalculator($orderId);
+
         $data = $table->getRowsHash();
+        $data['price_tax_incl'] = $data['price_tax_incl'] ?? (string) $taxCalculator->addTaxes($data['price']);
```

Comment:
Does it mean you can force the value by using `price_tax_incl` in the behat data? Then in that case you don't need the `$taxCalculator` Maybe you should handle this separately in an if statement rather than a ternary